TodoListClient/tests/acceptance/s/todo-items/index-test.js

```diff
@@ -0,0 +1,51 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { currentURL, findAll, visit } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import {
+  authenticateSession,
+  invalidateSession,
+} from 'ember-simple-auth/test-support';
+
+module('Acceptance | S | Todo Items | Index', (hooks) => {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('redirects to login if not authenticated', async function (assert) {
+    assert.expect(1);
+
+    await invalidateSession();
+    await visit('/s/todo-items');
+
+    assert.strictEqual(currentURL(), '/login', 'the route is correct');
+  });
+
+  test("shows list of user's todo items", async function (assert) {
```

Comment:
I would have expected an assertion on the outgoing HTTP request that fetches the todo-items somewhere.